Studies/source/MvaVariableSelection.cxx

```diff
@@ -120,36 +135,36 @@ class VariableDistribution {
             });
             const auto& best_entry = distance.front();
             for(const auto& name : best_entry.first) {
+                if (name == "mass_H" || name ==  "MX") continue;
```

Comment:
this condition shouldn't be here inside the algorithm. If needed, exclude them in the initialization step.